tor_ocr/main.py

```diff
@@ -260,6 +270,29 @@ def run(config):
 
         thing_to_reply_to = thing_to_reply_to.reply(_(chunk))
 
+        # grab first comment for transcription url
+        if counter==0:
+            first_comment = thing_to_reply_to
+
+    # Get submission from blossom, hard coded for testing purposes
+    blossom_submission = b_api.get("/submission/1/").json()
```

Comment:
Let's remove this and uncomment out the production version :)

Reply:
Done